pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -302,11 +310,9 @@ func updateCsiConfigMapOwnerRefs(ctx context.Context, namespace string, clientse
 // SaveClusterConfig updates the config map used to provide ceph-csi with
 // basic cluster configuration. The clusterNamespace and clusterInfo are
 // used to determine what "cluster" in the config map will be updated and
-// the clusterNamespace value is expected to match the clusterID
-// value that is provided to ceph-csi uses in the storage class.
 // The locker l is typically a mutex and is used to prevent the config
```

Comment:
Independently from this PR... Is "locker l" obsolete? I don't see it here anymore. Or it must be referring to the package var `configMutex`. Maybe we can just remove this comment.

Reply:
Yeah, that makes sense too 👍